static/js/tippyjs.js

```diff
@@ -163,19 +163,12 @@ export function initialize() {
         onShow(instance) {
             // Handle dynamic "starred messages" and "edit" widgets.
             const $elem = $(instance.reference);
-            let content = $elem.attr("data-tippy-content");
-            if (content === undefined) {
-                // Tippy cannot get the content for message edit button
-                // as it is dynamically inserted based on editability.
-                // So, we have to manually get the i element to get the
-                // content from it.
-                //
-                // TODO: Change the template structure so logic is unnecessary.
-                const $edit_button = $elem.find("i.edit_content_button");
-                content = $edit_button.attr("data-tippy-content");
-            }
 
-            instance.setContent(content);
+            if (!($elem.hasClass("edit_content") || $elem.hasClass("star_container"))) {
```

Comment:
Why is this the right conditional here? I'd hope we'd have a conditional that is more general than listing specific elements -- can we just check whether a `data-tooltip-template-id` element is present?

Reply:
Fixed, the current implementation doesn't require the conditional. The conditional statement was redundant, hence removed it with the latest commit.